conslogging/conslogging.go

```diff
@@ -222,17 +225,17 @@ func (cl ConsoleLogger) printPrefix() {
 		*cl.nextColorIndex = (*cl.nextColorIndex + 1) % len(availablePrefixColors)
 	}
 	c = cl.color(c)
-	c.Fprintf(cl.w, cl.prettyPrefix())
+	c.Fprintf(cl.outW, cl.prettyPrefix())
 	if cl.isFailed {
-		cl.w.Write([]byte(" *"))
-		cl.color(warnColor).Fprintf(cl.w, "failed")
-		cl.w.Write([]byte("*"))
+		cl.outW.Write([]byte(" *"))
```

Comment:
same comment as earlier regarding printing this failure to stderr too?

Reply:
Not sure what you mean here. I think printing the failed output to stderr could be interesting. 🤔

Reply:
I need to have calls to `Warnf()` go to stderr rather than stdout. Because without that change, I can't do:

```
MY_KEY=$(earth secrets get /user/secret_key)
```

I made the change in this PR to fix that issue, and I was wondering if we should also switch the errors that occur during a regular build to also go to stderr too. I'm fine leaving it as stdout as I just need to fix `warnf` to unblock progress with accessing secrets.

Reply:
Sounds good to me either way. No strong opinion.